src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/get_interior_vehicle_data_request.cc

```diff
@@ -377,12 +385,16 @@ void GetInteriorVehicleDataRequest::ProccessSubscription(
                                                       << module_type << " "
                                                       << module_id);
       extension->SubscribeToInteriorVehicleData(module);
+      app->UpdateHash();
     } else {
-      LOG4CXX_DEBUG(logger_,
-                    "UnsubscribeFromInteriorVehicleData "
-                        << app->app_id() << " " << module_type << " "
-                        << module_id);
-      extension->UnsubscribeFromInteriorVehicleData(module);
+      if (extension->IsSubscribedToInteriorVehicleData(module)) {
+        LOG4CXX_DEBUG(logger_,
+                      "UnsubscribeFromInteriorVehicleData "
+                          << app->app_id() << " [" << module_type << ":"
+                          << module_id << "]");
+        extension->UnsubscribeFromInteriorVehicleData(module);
+        app->UpdateHash();
```

Comment:
What is the use case of this hash update?

Reply:
propose to move app->UpdateHash(); into extension->UnsubscribeFromInteriorVehicleData and SubscribeFromInteriorVehicleData

Reply:
@LuxoftAKutsan Done in the 9e50bc725ce40f2caa8a2f5ed2e2f7287b02e2d8

Reply:
@IGapchuk And please remove UpdateHash() from this func